src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+import { roundedDown } from './utility/roundedDown';
 
-export class Triangle implements Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Circle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => {};
```

Comment:
[CODE STYLE] - The `getArea` method in the `Figure` interface should not specify any parameters, as all necessary values should be available as object fields.